internal/cluster/validator.go

```diff
@@ -509,3 +515,27 @@ func (v *clusterValidator) printNtpServerConfigured(c *clusterPreprocessContext,
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *clusterValidator) isSufficientNumberOfHosts(c *clusterPreprocessContext) validationStatus {
+
+	// nothing to validate in regular mode
+	if swag.StringValue(c.cluster.HighAvailabilityMode) != models.ClusterHighAvailabilityModeNone {
+		return ValidationSuccess
+	}
+
+	if len(c.cluster.Hosts) > 1 {
```

Comment:
We don't need to make sure it's eligible to be a master at least? maybe we need even better spec